### ISSUE TITLE
Prevent "document was modified" message when quitting if the document and the already saved one are the same

### ISSUE THREAD

**Issue by stefanoborini** [stefanoborini/vai#13]
If the user does operations, undo them, then quit, it will prevent a rapid quit because the document was modified. In reality, the document is the same. We need some sort of checksum to verify if the document has changed. One obvious strategy is this:

before quitting, check if modified.
if not, just quit
if yes, compute the md5 of the current document, and the md5 of the same file on disk.
if equal, just quit.
if not, refuse quitting without saving or enforcement.

One can check size first to improve performance.

depends on #64

**Comment by stefanoborini** [comment]
Closing as is. Now track the modified state across command execution. Will open new issue for checking md5.
